src/plugins/hetero/infer_request.cpp

```diff
@@ -112,7 +112,7 @@ const InferenceEngine::PreProcessInfo& HeteroInferRequest::GetPreProcess(const s
 
 void HeteroInferRequest::InferImpl() {
     for (auto&& desc : _inferRequests) {
-        OV_ITT_SCOPED_TASK(itt::domains::HeteroPlugin, desc._profilingTask);
+        OV_ITT_SCOPED_TASK(ov::hetero::itt::domains::Hetero, desc._profilingTask);
```

Comment:
Hetero 2 times...

Reply:
Fixed in https://github.com/openvinotoolkit/openvino/pull/18429